### DISCUSSION TITLE
Disable Underfull \hbox (badness 10000), but display Overfull \hbox (16.1262pt too wide)?

### DISCUSSION THREAD

**Discussion by HenningCode** [James-Yu/LaTeX-Workshop#3268] [Q&A]
Hello as the title suggests I want to disable the `Underfull \hbox (badness 10000)` message, but still display the `Overfull \hbox (16.1262pt too wide)` message, because that one is way more important to me. 

Underfull boxes only occur for me at least after I use `\\`, but Overfull if a word is to long and I may not realize that.

Greetings

**Comment by jlelong** [comment]
Use [`latex-workshop.message.latexlog.exclude
 `](https://github.com/James-Yu/LaTeX-Workshop/wiki/Compile#latex-workshopmessagelatexlogexclude) to drop underfull messages

```json
"latex-workshop.message.latexlog.exclude": [
    "^Underfull \\hbox"
]
```

You can also play with the badness value in your LaTeX files, see https://www.overleaf.com/learn/latex/%5Chbadness

**Reply by HenningCode** [reply]
This does not work for me sadly.

**Reply by jlelong** [reply]
If you modify the value of `latex-workshop.message.latexlog.exclude` using VS Code Preference, you can directly write
![Capture d’écran 2022-04-28 à 10 27 38](https://user-images.githubusercontent.com/2910140/165710900-8487c0ef-c241-4211-a27c-f9f48376f8fc.png)

Yet, if you edit the json file, you have to doubly escape the `\`

```json
"latex-workshop.message.latexlog.exclude": [
    "^Underfull \\\\hbox"
]
```
